modules/tensor_mechanics/src/actions/DomainIntegralAction.C

```diff
@@ -65,6 +65,16 @@ DomainIntegralAction::validParams()
       "displacements",
       "The displacements appropriate for the simulation geometry and coordinate system");
   params.addParam<VariableName>("temperature", "", "The temperature");
+  params.addParam<MaterialPropertyName>(
+      "functionally_graded_youngs_modulus_crack_dir_gradient",
+      "A grading parameter for cracks in functinonally graded materials (optional). Must be "
```

Comment:
This description is out of date. Suggestion:

`Gradient of the spatially varying Young's modulus provided in 'functionally_graded_youngs_modulus' in the direction of crack extension.`